### ISSUE TITLE
Reserved usernames

### ISSUE THREAD

**Issue by BronzW** [benawad/dogehouse#2389]
I feel like usernames like @elonmusk and @dogehouse should be reserved.

**Comment by A-Yamout** [comment]
@BronzW Are we going to have to make a list of all the names?

**Comment by BronzW** [comment]
If implemented, yes.

**Comment by dk-raw** [comment]
That's sounds like a good idea

**Comment by drvladb** [comment]
How would this list be sourced? What about people that already occupy these names?

**Comment by DevParapalli** [comment]
> I feel like usernames like @ElonMusk and @DogeHouse should be reserved.

How about a simple list of accounts that are **directly** related to DogeHouse, ie @DogeHouse, @DogeHouseBot etc, and an appeal system where "Moderators" or "Admins" are able to grant usernames to high-profile people if contacted. 
If @ElonMusk wanted to join, he would register a account with DogeHouse, send a appeal (with proof, maybe using work email) to the ??modmail?? that his username be changed to @ElonMusk if someone else was registered with it.

**Comment by BronzW** [comment]
> How would this list be sourced? What about people that already occupy these names?

Their username would be changed.

**Comment by younesaassila** [comment]
> 
> 
> > I feel like usernames like @ElonMusk and @DogeHouse should be reserved.
> 
> How about a simple list of accounts that are **directly** related to DogeHouse, ie @DogeHouse, @dogehousebot etc, and an appeal system where "Moderators" or "Admins" are able to grant usernames to high-profile people if contacted.
> If @ElonMusk wanted to join, he would register a account with DogeHouse, send a appeal (with proof, maybe using work email) to the ??modmail?? that his username be changed to @ElonMusk if someone else was registered with it.

How about automatically authorizing someone of using one of these usernames if they are the owner of it on Twitter/GitHub? That way if Elon Musk were to join DogeHouse, there would not be a need for him to send anything as he could just log in using his Twitter account @elonmusk.

PS: I feel like all verified Twitter accounts should have their username reserved.

**Comment by BronzW** [comment]
> > > I feel like usernames like @ElonMusk and @DogeHouse should be reserved.
> > 
> > 
> > How about a simple list of accounts that are **directly** related to DogeHouse, ie @DogeHouse, @dogehousebot etc, and an appeal system where "Moderators" or "Admins" are able to grant usernames to high-profile people if contacted.
> > If @ElonMusk wanted to join, he would register a account with DogeHouse, send a appeal (with proof, maybe using work email) to the ??modmail?? that his username be changed to @ElonMusk if someone else was registered with it.
> 
> How about automatically authorizing someone of using one of these usernames if they are the owner of it on Twitter/GitHub? That way if Elon Musk were to join DogeHouse, there would not be a need for him to send anything as he could just log in using his Twitter account @ElonMusk.
> 
> PS: I feel like all verified Twitter accounts should have their username reserved.

Yeah definitely since DogeHouse uses twitter for it's login system.

**Comment by A-Yamout** [comment]
Could another idea be to let people use any username, and then make then change it for "impersonation" of people if it's on a list without proof?

**Comment by BronzW** [comment]
Works too
